core/src/main/resources/org/apache/brooklyn/core/mgmt/persist/deserializingClassRenames.properties

```diff
@@ -1434,3 +1434,5 @@ brooklyn.test.entity.BlockingEntityImpl
 org.apache.brooklyn.config.ConfigInheritance$None                                : org.apache.brooklyn.config.ConfigInheritance$Legacy$None
 org.apache.brooklyn.config.ConfigInheritance$Always                              : org.apache.brooklyn.config.ConfigInheritance$Legacy$Always
 org.apache.brooklyn.config.ConfigInheritance$Merged                              : org.apache.brooklyn.config.ConfigInheritance$Legacy$Merged
+
+org.apache.brooklyn.entity.software.base.lifecycle.MachineLifecycleEffectorTasks$ProvisioningTaskState : org.apache.brooklyn.core.entity.internal.AttributesInternal$ProvisioningTaskState
```

Comment:
Can you add a comment "since 0.10.0" so we can start removing entries at some point.